authorization-service/src/functions/basic-authorizer/handler.ts

```diff
@@ -0,0 +1,27 @@
+import { middyfy } from '@libs/lambda';
+import { AuthorizationService } from 'src/services/authorization.service';
+
+const authorizerType = 'TOKEN';
+
+const basicAuthorizer = async (event) => {
+	const { authToken = '', methodArn } = event;
+	const authorizerService = new AuthorizationService();
+
+	if (event?.type !== authorizerType) {
+		return authorizerService.generatePolicy(authToken, methodArn, 'Deny');
+	}
+
+	try {
+		const policy = authorizerService.generatePolicy(
```

Comment:
The third parameter of this method is string with value either 'Deny' or 'Allow'. Why do you pass to it event.methodArn instead?